opentelemetry-sdk/src/metrics/periodic_reader.rs

```diff
@@ -264,20 +263,26 @@ impl<RT: Runtime> PeriodicReaderWorker<RT> {
         match message {
             Message::Export => {
                 if let Err(err) = self.collect_and_export().await {
-                    global::handle_error(err)
+                    otel_error!(name: "PeriodicReaderWorker.Export.Error",error = err);
```

Comment:
you can ignore this file initially, to avoid conflict for my PR on the same file.